src/Agent/NewRelic/Profiler/Configuration/InstrumentationConfiguration.h

```diff
@@ -394,6 +388,25 @@ namespace NewRelic { namespace Profiler { namespace Configuration
             return std::unique_ptr<xstring_t>(new xstring_t(attribute->value())); 
         }
 
+        static std::unique_ptr<xstring_t> NormalizeParameters(std::unique_ptr<xstring_t> rawParams)
+        {
+            if (rawParams == nullptr)
+            {
+                return nullptr;
+            }
+
+            // void as the parameters means parameterless method call (not all overloads)
+            if (Strings::AreEqualCaseInsensitive(*(rawParams), _X("void")))
```

Comment:
This is not related to this PR but rather interesting to me. I don't know if `void` parameter is a thing in C#. It is a thing in C++ or C but we are instrumenting C# functions.

Reply:
I actually found my answer. Whatever method signatures we declare in the instrumetation xml are IL method signatures as opposed to C# or languages. So I believe `void` parameter is a thing in IL world.